var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -8,12 +8,36 @@
 
 from spack.package import *
 
+rocm_dependencies = [
+    "hip",
+    "rocrand",
+    "rocblas",
+    "rocfft",
+    "hipfft",
+    "rccl",
+    "hipsparse",
+    "rocprim",
+    "llvm-amdgpu",
+    "hsa-rocr-dev",
+    "rocminfo",
+    "hipsolver",
+    "hiprand",
+    "rocsolver",
+    "hipsolver",
+    "hipblas",
+    "hipcub",
+    "rocm-core",
+    "roctracer-dev",
+    "miopen-hip",
+]
+
 
 class PyTensorflow(Package, CudaPackage, ROCmPackage, PythonExtension):
     """TensorFlow is an open source machine learning framework for everyone."""
 
     homepage = "https://www.tensorflow.org"
     url = "https://github.com/tensorflow/tensorflow/archive/v2.3.1.tar.gz"
+    git = "https://github.com/ROCm/tensorflow-upstream.git"
```

Comment:
Let's use the official git here instead of this fork

Reply:
Unfortunately, the branch we're using is only available on the the ROCm fork of tensorflow.

Reply:
You can add a version-specific git arg instead of using the global class attribute.

Reply:
done, thanks!